tests/jballerina-unit-test/src/test/resources/test-src/types/byte/byte_array_value.bal

```diff
@@ -33,6 +36,26 @@ function base64ByteArrayLiteralWithWS() {
     assertEquality(byteArray1, byteArray3);
 }
 
+function testByteArrayLiteralInherentType() {
+    var byteArray1 = base16 `aa bb`;
```

Comment:
Do we have compile-time tests like
```ballerina
    byte[2] _ = byteArray1;
```

Reply:
Updated a41828e66acf222373ef93deff481bde6310fd77